README.md

```diff
@@ -74,7 +74,7 @@ Check out the full [reference of AuroraBoot  in our documentation](https://kairo
 For instance, in one machine from your workstation, you can run:
 
 ```bash
-$ docker run --rm -ti --net host quay.io/kairos/auroraboot --set "artifact_version=v1.5.0" --set "release_version=v1.5.0" --set "flavor=rockylinux" --set repository="kairos-io/kairos" --cloud-config /....
+$ docker run --rm -ti --net host quay.io/kairos/auroraboot --set "artifact_version=v2.4.2" --set "release_version=v2.4.2" --set "flavor=rockylinux"--set "flavor_release=9"  --set repository="kairos-io/kairos" --cloud-config /....
```

Comment:
shouldn't this be "flavor_version" like below?

Reply:
or even better "flavor_release" everywhere so that we are consistent with Kairos repo?

Reply:
shoot, good catch